docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+View and reset consumer group offsets
+============================================
+
+The `open source Apache Kafka code <https://kafka.apache.org/downloads>`_ includes a ``kafka-consumer-groups.sh`` utility enabling you to view and manipulate the state of consumer groups. 
+Before using the tool, few setup steps need to be performed.
+
+Define the configuration file
+-----------------------------
+
+#. Create the Java keystore and truststore for your Aiven for Apache Kafka service using the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`.
```

Comment:
We have another article explaining how to create keystore and truststore in more details! I think we should link it here  once it is merged :)

Reply:
Agreed, let's wait to merge that one :D